### PR TITLE
added seperate spacs js file

### PR DESCRIPTION
**Pull request by notjustinshaw**
This will add the blubbr.io/spacs page to the website as well as add some well-needed project stucture to this react app.

### CODE REVIEW

**Comment by sorenrood** [comment]
@justinalexandershaw I'll check this out tonight. Working on some homework atm.

**Comment by notjustinshaw** [comment]
> @justinalexandershaw I'll check this out tonight. Working on some homework atm.

Great yeah just lmk, I'll try to get it green before we merge it.

File: src/components/PageRouter.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// to serve exactly one route at a time and switching over them is a good way to do that.
+function PageRouter() {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
+        <Route path="/spacs">
+          {/* <Spacs /> */}
```
Comment (sorenrood) [inline src/components/PageRouter.js]: Is there supposed to be something in here?

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import logo from "../assets/img/logo.svg";
+
+function Home() {
+  return (
+    <div>
+      <img src={logo} className="App-logo" alt="logo" />
+      <p>
+        Welcome to <code>blubbr.io</code>, Soren Rood!
```
Comment (sorenrood) [inline src/pages/Home.js]: epic

**Review by sorenrood — DISMISSED** [review]
After it builds, go ahead and merge. 🔥

**Comment by notjustinshaw** [comment]
Okay I got everything working. We now have a blubbr.io homepage and a blubbr.io/spacs page which has steven's embed from tableau. Once the tests turn green I will go ahead and merge this into master @sorenrood

**Comment by sorenrood** [comment]
Awesome job. Can't wait to see it! :smile: :fire:

**Comment by sorenrood** [comment]
Hmm, looks like it failed to merge @justinalexandershaw.

image

**Comment by notjustinshaw** [comment]
@sorenrood yeah looks like it. huh. I've never seen this error before

**Comment by sorenrood** [comment]
> @sorenrood yeah looks like it. huh. I've never seen this error before

Yeah, haven't seen it either.

**Comment by notjustinshaw** [comment]
Okay I tried [this](https://stackoverflow.com/a/52106392/12849203) and we will have to wait and see if that works.

**Comment by sorenrood** [comment]
There we go @justinalexandershaw! Passing! :fire:

**Review by sorenrood — APPROVED** [review]
Looks like it built!

**Comment by sorenrood** [comment]
Ok- now I'm confused. How did someone push a commit without signing? Hmm. 

image

**Comment by sorenrood** [comment]
@justinalexandershaw I still have the option to force merge if you want me to.

**Comment by notjustinshaw** [comment]
Yeah so do I -- let me try to figure out why its blocked

**Comment by notjustinshaw** [comment]
LGTM

**Pull request merged**